vmdb/app/models/job.rb

```diff
@@ -55,7 +55,7 @@ def check_active_on_destroy
 
   def self.agent_state_update_queue(jobid, state, message=nil)
     begin
-      job = Job.find(:first, :conditions => ["guid = ?", jobid], :select => "id, state, guid")
+      job = Job.where("guid = ?", jobid).select("id, state, guid").first
```

Comment:
No, there is only one situation that doesn't work:  select with multiple arguments (the referenced code above has a single string with comma separated values):

```ruby

irb(main):006:0> MiqProductFeature.select("id, name").take(2)
=> [#<MiqProductFeature id: 1, name: "Everything">, #<MiqProductFeature id: 2, name: "Workloads">]

irb(main):008:0> MiqProductFeature.select(["id", "name"]).take(2)
=> [#<MiqProductFeature id: 1, name: "Everything">, #<MiqProductFeature id: 2, name: "Workloads">]

irb(main):009:0> MiqProductFeature.select([:id, :name]).take(2)
=> [#<MiqProductFeature id: 1, name: "Everything">, #<MiqProductFeature id: 2, name: "Workloads">]

```

This fails:
```ruby
irb(main):010:0> MiqProductFeature.select(:id, :name).take(2)
ArgumentError: wrong number of arguments (2 for 0..1)

irb(main):011:0> MiqProductFeature.select('id', 'name').take(2)
ArgumentError: wrong number of arguments (2 for 0..1)
```

The confusing part is that I believe the other arel methods like order, includes, etc. accept multiple arguments (this latter failing case).